app/controllers/catalog_controller.rb

```diff
@@ -53,6 +53,11 @@ class CatalogController < ApplicationController
     config.add_show_field "tag_ssim", label: "Tags", link_to_facet: true
     config.add_show_field SolrDocument::FIELD_WORKFLOW_ERRORS, label: "Error", helper_method: :value_for_wf_error
 
+    # exploded_project_tag_ssim indexes all project tag prefixes whereas project tag_ssim only indexes whole tags.
+    # we want to facet on exploded_project_ag_ssim to get the hierarchy.
```

Comment:
typo? `exploded_project_ag_ssim` -> `exploded_project_tag_ssim`

Reply:
not having to do with agriculture, i fixed this